advanced/Scripts/chronometer.sh

```diff
@@ -297,30 +369,71 @@ get_ftl_stats() {
   dns_queries_today_raw="${stats_raw[3]}"
   ads_blocked_today_raw="${stats_raw[5]}"
   ads_percentage_today_raw="${stats_raw[7]}"
-
+  queries_forwarded_raw="${stats_raw[11]}"
+  queries_cached_raw="${stats_raw[13]}"
+  
   # Only retrieve these stats when not called from jsonFunc
   if [[ -z "$1" ]]; then
-    local recent_blocked_raw
     local top_ad_raw
     local top_domain_raw
     local top_client_raw
-  
-    domains_being_blocked=$(printf "%'.0f\n" "${domains_being_blocked_raw}")
-    dns_queries_today=$(printf "%'.0f\n" "${dns_queries_today_raw}")
-    ads_blocked_today=$(printf "%'.0f\n" "${ads_blocked_today_raw}")
-    ads_percentage_today=$(printf "%'.0f\n" "${ads_percentage_today_raw}")
     
-    recent_blocked_raw=$(pihole-FTL recentBlocked)
+    domains_being_blocked=$(printf "%.0f\n" "${domains_being_blocked_raw}")
+    dns_queries_today=$(printf "%.0f\n" "${dns_queries_today_raw}")
+    ads_blocked_today=$(printf "%.0f\n" "${ads_blocked_today_raw}")
+    ads_percentage_today=$(printf "%'.0f\n" "${ads_percentage_today_raw}")
+    queries_cached_percentage=$(printf "%.0f\n" "$(calcFunc "$queries_cached_raw * 100 / ( $queries_forwarded_raw + $queries_cached_raw )")")
+    recent_blocked=$(pihole-FTL recentBlocked)
     top_ad_raw=($(pihole-FTL "top-ads (1)"))
```

Comment:
I wonder if you should quote here to avoid splitting (see also the next two lines).
`array=( "$(mycommand)" )` as the output should be a single element (and there is no fail check here).